### PR TITLE
test commit

### PR DESCRIPTION
**Pull request by ray4spd**

### CODE REVIEW

**Comment by calebpower** [comment]
What does this test commit do? Is it the application of the program stub?

File: README.md [hunk 1]
```diff
@@ -1,2 +1,13 @@
...
+recource: https://www.geeksforgeeks.org/python-convert-speech-to-text-and-text-to-speech/<br>
+Python requirments<br>
+- python3.6
+- pip3.6
+
+pip commands for windows:<br>
+`$ pip install speechrecognition` <br>
+`$ pip install pyaudio`<br>
+`$ pip install playsound`<br>
+`$ pip install pocketsphinx` <br>
```
Comment (calebpower) [inline README.md]: Can you add a part that tells me how to run the program?

File: Recognizer/desktop.ini [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[LocalizedFileNames]
+chord.wav=@%windir%\system32\mmres.dll,-701
+ding.wav=@%windir%\system32\mmres.dll,-702
```
Comment (calebpower) [inline Recognizer/desktop.ini]: This is a junk file... `desktop.ini` is auto-generated by Windows and should be added to the `.gitignore` file.

**Review by calebpower — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ray4spd) [reply README.md]: Added how to run

File: Recognizer/desktop.ini (hunk 2 above)
Comment (ray4spd) [reply Recognizer/desktop.ini]: Removed file and added .gitignore

**Review by calebpower — APPROVED** [review]
Great work Ray!

**Pull request merged**